.travis.yml

```diff
@@ -55,6 +55,11 @@ jobs:
         - IOS_BUILD_TYPE='Debug'
```

Comment:
let's rename the job to something like `Debug / Tests / Simulator / Xcode 11.2`

Reply:
What about
`iOS / Debug (Tests) / Simulator / Xcode 11.2`
instead of
`iOS / Debug / Simulator / Xcode 11.2`?